src/assets/styles/layout.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 .container {
-  max-width: 1360px; // should match auth-web, etc
```

Comment:
This was a recent fix that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit